### PR TITLE
Explain how to create scroll effects

### PR DESCRIPTION
**Pull request by blasten**
This is for: https://github.com/PolymerLabs/app-layout/issues/135

### CODE REVIEW

File: app-scroll-effects/README.md [hunk 1]
```diff
@@ -1,35 +1,75 @@
...
+<app-header effects="waterfall"></app-header>
+```
+
+### Creating scroll effects
+
+You may want to create a custom scroll effect if you need to modify the CSS of an element
+based on the scroll position.
+
+A scroll effect definition is an object with a `setUp()`, `tearDown()` and `run()` function.
+The event name is the used as the key to register the definition in the `Polymer.AppLayout.scrollEffects`
```
Comment (frankiefu) [inline app-scroll-effects/README.md]: The event name is ~~the~~ used as the key to register

**Comment by frankiefu** [comment]
minor nit, otherwise LGTM. Thanks.

**Comment by keanulee** [comment]
FYI if you include in the PR description:

Fixes #135 

Then GitHub will automatically close the issue once it has been merged.

File: app-scroll-effects/README.md [hunk 2]
```diff
@@ -1,35 +1,75 @@
-## Polymer.AppScrollEffectsBehavior
+## Scroll effects
 
-`Polymer.AppScrollEffectsBehavior` provides an interface that allows to consume scrolls events.
-Effects are installed in the `Polymer.AppLayout.scrollEffects` namescape.
+`Polymer.AppScrollEffectsBehavior` provides an interface that allows to consume scrolls effects.
```
Comment (keanulee) [inline app-scroll-effects/README.md]: ...allows an element to use...

File: app-scroll-effects/README.md (hunk 1 above)
Comment (blasten) [inline app-scroll-effects/README.md]: "effect"

File: app-scroll-effects/README.md [hunk 3]
```diff
@@ -1,35 +1,75 @@
...
-<x-element effects="waterfall"></x-element>
+<app-header effects="waterfall"></app-header>
+```
+
+### Creating scroll effects
+
+You may want to create a custom scroll effect if you need to modify the CSS of an element
+based on the scroll position.
+
+A scroll effect definition is an object with a `setUp()`, `tearDown()` and `run()` function.
```
Comment (keanulee) [inline app-scroll-effects/README.md]: ...with `setUp()`, `tearDown()` and `run()` functions.

(Or replace functions with methods, but make sure it's plural)

File: app-scroll-effects/README.md [hunk 4]
```diff
@@ -1,35 +1,75 @@
...
+The event name is the used as the key to register the definition in the `Polymer.AppLayout.scrollEffects`
+namespace. For example, let's define an effect that resizes the header's logo:
+
+```js
+Polymer.AppLayout.scrollEffects.logo = {
+  setUp: function(config) {
```
Comment (keanulee) [inline app-scroll-effects/README.md]: May be helpful to explain what the `config` argument does.

Also fine if you want to exclude it from the API, but then it should be removed here in this function definition.

Comment (blasten) [inline app-scroll-effects/README.md]: that is already covered https://github.com/PolymerLabs/app-layout/tree/master/app-scroll-effects#configuring-effects

**Comment by keanulee** [comment]
LGTM

**Pull request merged**